ckanext/unaids/react/components/FileInputComponent/src/App.js

```diff
@@ -16,9 +23,15 @@ export default function App({ loadingHtml, maxResourceSize, lfsServer, orgId, da
     const [hiddenInputs, _setHiddenInputs] = useState({});
     const [uploadError, setUploadError] = useState(false);
     const [useEffectCompleted, setUseEffectCompleted] = useState(false);
+    const [selectedResource, setSelectedResource] = useState({
+        resource: null,
+        dataset: null,
+    });
 
     const setHiddenInputs = (newUploadMode, metadata) => {
         setUploadMode(newUploadMode);
+        let fileFormatField =
+                        document.getElementById('field-format');
```

Comment:
we have a line size limit set to 127. No need to break it here.

Reply:
Where is this set for JS/TS?

Reply:
Actually... is this even set somewhere for Python inside the repo? I don't see a .flake8 or a [flake8] section in the pyproject.toml?

Reply:
we have the limit hardcoded in flake8 command inside github actions config.